### ISSUE TITLE
Template reference not found when call Register api

### ISSUE THREAD

**Issue by megataps** [Sylius/ShopApiPlugin#707]
I'm using setup Sylius on my local and then install this plug in. 
I have tried to use register api but I got this error.
`{
    "code": 500,
    "message": "Template reference \"SyliusShopApiPlugin:Email:verification.html.twig\" not found, did you mean \"@SyliusShopApiPlugin/Email/verification.html.twig\"?"
}
`

<img width="880" alt="Screen Shot 2021-03-29 at 22 04 25" src="https://user-images.githubusercontent.com/7032500/112857204-cfe39180-90da-11eb-9445-5156147db074.png">

**Comment by mamazu** [comment]
Yeah seems like some parts of the plugin still use the old Template syntax.

**Comment by megataps** [comment]
I have clear the project and then install the new one.

1. I have followed this instruction to install Sylius:
`
composer create-project sylius/sylius-standard acme
cd acme
php bin/console sylius:install
yarn install
yarn run gulp
symfony server:start
open http://127.0.0.1:8000
`
2. Then I have follow this link:  [Sylius Documentation - How to disable Sylius shop?](https://docs.sylius.com/en/1.6/cookbook/configuration/disabling-shop-admin-api.html#how-to-disable-sylius-shop)

But the error still the same. I don't know what is wrong?

**Comment by mamazu** [comment]
Can you run the command
`bin/console debug:config sylius_mailer`
and check if in the output you also find the invalid template paths?

**Comment by megataps** [comment]
Hi @mamazu , When do we have the next release for this iusses?

**Comment by mamazu** [comment]
We are currently working on some concurrency issues then we will release a new version.

**Comment by mamazu** [comment]
Version 1.4 was released. Maybe check again if the error still occurs.
